server/graphql/v2/mutation/MemberMutations.js

```diff
@@ -0,0 +1,56 @@
+import { GraphQLNonNull, GraphQLString } from 'graphql';
+
+import models from '../../../models';
+import { NotFound, Unauthorized } from '../../errors';
+import { idDecode } from '../identifiers';
+import { AccountReferenceInput } from '../input/AccountReferenceInput';
+import { Member } from '../object/Member';
+
+const memberMutations = {
+  editPublicMessage: {
+    type: new GraphQLNonNull(Member),
+    description: 'Edit the public message for the given Member of a Collective',
+    args: {
+      FromCollectiveLegacyId: {
+        type: AccountReferenceInput,
+        description: 'Legacy id for the donating Collective',
+      },
+      ToCollectiveId: {
```

Comment:
Should be `toAccount`, and type should be non-nullable.

Also `description` doesn't match the `type`.